DesignPattern/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace DesignPattern
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            WheatPizza wheatPizza;
+            WhitePizza whitePizza;
+
+            WheatPizzaBuilder wheatPizzaBuilder = new WheatPizzaBuilder();
+            WhitePizzaBuilder whitePizzaBuilder = new WhitePizzaBuilder();
+
+            Director wheatPizzaDirector = new Director(wheatPizzaBuilder);
```

Comment:
You don't need to declare another Director, one Director can work with any builder